dtfs-central-api/src/v1/swagger-definitions/portal/facility-bss.js

```diff
@@ -16,8 +16,8 @@
  *       createdDate:
  *         type: string
  *         example: '1632392646415'
- *       lastEdited:
- *         type: string
+ *       updatedAt:
+ *         type: integer
  *         example: '1632392778918'
```

Comment:
minor thing:
if type is integer, then we can remove the quotes on this one, right?

Reply:
Great spot, fixed